My_Clean-Architecture/Domain/Entities/Dog.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  Dog.swift
+//  My_Clean-Architecture
+//
+//  Created by OWEN on 9/20/24.
+//
+
+import Foundation
+
+struct Dog: Codable {
+    let stringURL: String
```

Comment:
도메인 모델의 변수명은 의미 전달이 좀 더 잘되면 좋을거 같아요
예를 들면 아래처럼이요~
```suggestion
    let imageURL: String
```